lib/perl5db.pl

```diff
@@ -1223,8 +1223,19 @@ =head2 SETTING UP THE DEBUGGER GREETING
 
 # Sets up $emacs as a synonym for $client_editor.
 our ($client_editor);
+if ($ENV{EMACS}) { print STDERR "ZZZ: EMACS\n"; }
 *emacs = $client_editor if $client_editor;    # May be used in afterinit()...
 
+package RenameVariable;
+require Tie::Scalar;
+our @ISA = qw(Tie::Scalar);
+our $message = '$slave_editor deprecated; use $client_editor instead';
+sub FETCH { warn $message; }
```

Comment:
This should probably return $client_editor

Reply:
Thanks.  p.r. updated.